src/components/Authorization/Authorization.css

```diff
@@ -14,9 +10,13 @@ input::placeholder {
   border-color: #282828;
```

Comment:
Better use .scss and variables $primary-dark-text

Reply:
fix